src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms linear;
```

specify properties you want to animate, + linear is default, no need to add it

```suggestion
    transition: color 300ms, transform 300ms;
```